spotrec.py

```diff
@@ -39,6 +39,7 @@
 _filename_pattern = "{trackNumber} - {artist} - {title}"
 _tmp_file = True
 _underscored_filenames = False
+_use_intern_track_counter = False
```

Comment:
minor one: meaning of "intern": "a student or trainee who works, sometimes without pay, at a trade or occupation in order to gain work experience."

guess "internal" is the correct term :D

Reply:
Oh yes, thanks. I am german and will never learn it... :-D

Reply:
Done with latest commit.

Reply:
Heißt bei uns btw. sowas wie Praktikant, was hier keinen Sinn ergibt. An sich hab ich nichts gegen Abkürzungen, aber sie müssen eindeutig sein :D